00-matching-logic.mm0

```diff
@@ -149,6 +149,7 @@ axiom sFresh_appCtx_box {box: SVar} (ctx psi: Pattern box):
 -- Definition of substitutions and application contexts as meta-level relations
 term Norm: Pattern > Pattern > Normalization;
 
+axiom eSubstitution_trivial {x: EVar} (phi: Pattern x): $ Norm (e[ eVar x / x ] phi) phi $;
```

Comment:
Maybe `eSubstitution_same_var`? `trivial` isn't very descriptive.

Reply:
That sounds too similar too `eSubstitution_in_same_eVar` and it would become confusing

Reply:
Actually, `eSubstitution_id`?